llvm/tools/llvm-cas-test/Config.def

```diff
@@ -0,0 +1,18 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// Define CONFIG before including.
+//
+// #define CONFIG(NAME, TYPE, DEFAULT_VAL, MIN_VAL, MAX_VAL, OPT_NAME, OPT_DESC)
+
+CONFIG(NumShards, uint8_t, 10, 1, 20, num-shards, "number of shards")
+CONFIG(NumChildren, uint8_t, 8, 1, 32, num-children, "number of child nodes")
+CONFIG(TreeDepth, uint8_t, 4, 0, 8, tree-depth, "tree depth")
+CONFIG(DataLength, uint16_t, 1, 1024, 4096, data-length, "data length")
+CONFIG(PrecentFile, uint8_t, 0, 10, 100, precent-file,
```

Comment:
DataLength and PercentFIle appear to be using [min, default, max] ordering, which differs from every other definition